ui/src/shared/components/DeleteDataForm/DeleteDataForm.tsx

```diff
@@ -16,23 +16,28 @@ import {Filter, RemoteDataState} from 'src/types'
 
 // Selectors
 import {setCanDelete} from 'src/shared/selectors/canDelete'
+// import {setCanDelete} from 'src/shared/selectors/canDelete'
 
-// action
+// Actions
 import {
   deleteFilter,
   deleteWithPredicate,
+  resetFilters,
   setBucketName,
   setDeletionStatus,
   setFilter,
   setIsSerious,
   setTimeRange,
 } from 'src/shared/actions/predicates'
+import {selectBucket} from 'src/timeMachine/actions/queryBuilder'
```

Comment:
So, it looks like the DeleteDataForm is using and editing state that the query Builder uses. We should probably have separate redux state for the delete data form even though it does similar things. :/

Reply:
Totally makes sense